launch-or-update-local.sh

```diff
@@ -4,14 +4,16 @@ function displayHelp {
   printf "usage: ./launch-or-update-local.sh [OPTIONS]
           -h, --help
             Display help
+          -l, --latest [true|false]
+            Set repo to latest dev versions (defaults to true)
           -u, --update [true|false]
             Update Helm Charts (defaults to true)
           "
   exit
 }
 # Option strings
-SHORT="h:u:"
-LONG="help,update:"
+SHORT="h:l:u:"
+LONG="help:,latest:,update:"
```

Comment:
think we need to remove the ':' so that the help flag doesn't expect an argument
`SHORT="h:l:u:"` -> `SHORT="hl:u:"` &
`LONG="help:,latest:,update:"` -> `LONG="help,latest:,update:"`

otherwise we won't be able to display the help by running `./launch-or-update-local.sh -h`

Reply:
Updated with correct Short/Long